pkg/qospolicy/apis/aci.qos/v1/types.go

```diff
@@ -0,0 +1,66 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type PolicyState string
+
+const (
+	Ready  PolicyState = "Ready"
+	Failed PolicyState = "Failed"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// QosPolicySpec defines the desired state of QosPolicy
+type QosPolicySpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	Selector PodSelector  `json:"selector,omitempty"`
+	Ingress  PolicingType `json:"ingress,omitempty"`
+	Egress   PolicingType `json:"egress,omitempty"`
+}
+
+//PolicingType
+type PolicingType struct {
+	PolicingRate  int `json:"policing_rate"`
+	PolicingBurst int `json:"policing_burst"`
+}
+
+// QosPolicyStatus defines the observed state of QosPolicy
+type QosPolicyStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file7
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	State PolicyState `json:"state"`
```

Comment:
Where are you setting your policy state in code? are you planning to use it for future ?

Reply:
yes...planning to use in future